lib/devices.js

```diff
@@ -114,7 +114,7 @@ module.exports = (log, db, push) => {
             deviceName = synthesizeName(deviceInfo)
           }
           if (credentials.tokenVerified) {
-            request.app.devices.then(devices => {
+            db.devices(credentials.uid).then(devices => {
```

Comment:
Why was this needed? I wonder if we can find a way to fix `request.app.devices` so that we maintain the nice caching behaviour.

Reply:
it's problematic, https://github.com/mozilla/fxa-auth-server/issues/2992 was filed.